production/ksonnet/loki/query-scheduler.libsonnet

```diff
@@ -0,0 +1,58 @@
+
+local k = import 'ksonnet-util/kausal.libsonnet';
+
+{
+  local container = k.core.v1.container,
+
+  // Override frontend and querier configuration
+  local override = {
```

Comment:
We'll need an enablement flag for parity with how we enable other optional components.